src/content/visualise/TabCheckboxDropdown.tsx

```diff
@@ -0,0 +1,78 @@
+﻿import { Box, Divider, Tab, Tabs, Typography } from '@mui/material'
+import React, { useEffect, useState } from 'react'
+import CheckboxDropdown from './CheckboxDropdown'
+
+interface TabPanelProps {
+  children?: React.ReactNode
+  index: number
+  value: number
+}
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props
+
+  return (
+    <div
+      role='tabpanel'
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box sx={{ p: 3 }}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  )
+}
+
+function a11yProps(index: number) {
```

Comment:
This seems like a strange name for a function. Can you please explain it, or otherwise change it to something a bit more descriptive (e.g. something like `tabProps` or `tabPropsGeneric`)?

Reply:
@RohitRajeshh